go-controller/pkg/ovn/ovn.go

```diff
@@ -63,6 +66,9 @@ type namespaceInfo struct {
 	// the policy itself.
 	networkPolicies map[string]*namespacePolicy
 
+	//defines the namespace egressFirewallRules
```

Comment:
This comment is sooo not matching the field name :D

Reply:
done